x-pack/plugins/ml/server/models/job_validation/job_validation.ts

```diff
@@ -100,6 +100,14 @@ export async function validateJob(
           ...(await validateModelMemoryLimit(client, mlClient, job, duration))
         );
       }
+
+      // if datafeed has aggregation, require job config to include a valid summary_doc_field_name
+      const datafeedAggConfig = job.datafeed_config?.aggregations ?? job.datafeed_config?.aggs;
+      if (datafeedAggConfig !== undefined) {
+        if (!job.analysis_config?.summary_count_field_name) {
```

Comment:
This could be combined into one `if` statement.

Reply:
Updated here https://github.com/elastic/kibana/pull/81923/commits/d96a119950e710b043eb3c89568be037cc203ed2